tests/test_search_schema.py

```diff
@@ -117,14 +117,26 @@ def testRcsbAttrs(self):
             attr_details = attrs.get_attribute_details("foo")
             self.assertIsNone(attr_details)
 
+    def testNestedAttrs(self):
+        with self.subTest(msg="3. Check nested attribute indexing dictionary length"):
+            nested_dict = SEARCH_SCHEMA.nested_attribute_schema
+            self.assertGreaterEqual(len(nested_dict), 15)
+
+        with self.subTest(msg="3. Check for a specific nested attribute"):
+            expected_tuple = ('rcsb_uniprot_annotation.name', 'rcsb_uniprot_annotation.type')
+            self.assertIn(expected_tuple, SEARCH_SCHEMA.nested_attribute_schema)
+
+            not_expected_tuple = ('rcsb_uniprot_annotation.name.asdlkfjaskdjfaskldjf', 'rcsb_uniprot_annotation.type')
```

Comment:
I appreciate the creativity lol, but can you use a pair of real attributes (but which _do not_ have nested context) for this check?